### DISCUSSION TITLE
Where to put Lycoris and how do I access them?

### DISCUSSION THREAD

**Discussion by tranzmatt** [vladmandic/sdnext#2403] [General]
With the new unified Lora, I'm not sure where to put my Lycoris.  I have a copy in 

`models/Lora/GoodHands-beta2.safetensors`

and reference it with

`<lyco:GoodHands-beta2:0.8>`

but it always says:

`Couldn't find Lora with name GoodHands-beta2`

What am I doing wrong?  This used to work find when there was a separate LyCORIS path.

**Comment by vladmandic** [comment]
just put everything in lora folder and reference with `<lora:xxx>`, it will figure what is what :)
`<lyco:xxx>` is obsolete, its stated in the changelog.

**Reply by tranzmatt** [reply]
Cool that works. Mixing loras and lycos violates my software dev sensibilities, but I can't argue the results ;)

**Reply by vladmandic** [reply]
there are too many networks nowadays, having separate folder or handler for each would be too much for normal user.

fyi:
> network_full.py  network_hada.py  network_ia3.py  network_lokr.py  network_lora.py  network_norm.py  network_oft.py

**Reply by tranzmatt** [reply]
I hadn't realized it had gotten so crazy.  I appreciate the work you've done then. :)
